league_table/__init__.py

```diff
@@ -1 +0,0 @@
-
```

Comment:
I have a feeling this is actually needed?

Reply:
@DavidLemayian - The `__init__.py` file is blank on the current commits. Hyphen is not present.

Reply:
@tebajanga thought it had been entirely deleted. Looks like it only removed the line.